maestro

```diff
@@ -84,20 +84,26 @@ class MaestroMonitor(object):
                 listen = []
                 for ep_name in self.daemons[dmn_grp][dmn_name]['endpoints']:
                     ep = self.daemons[dmn_grp][dmn_name]['endpoints'][ep_name]
+                    hostname = self.daemons[dmn_grp][dmn_name]['addr']
                     if 'maestro_comm' not in ep:
                         listen.append(ep)
                         continue
                     conf = check_opt('conf', ep)
-                    comm = Communicator(ep['xprt'],
-                                        ep['addr'],
-                                        ep['port'],
-                                        ep['auth']['name'],
-                                        { 'conf' : conf })
+                    try:
+                        comm = Communicator(ep['xprt'],
+                                            hostname,
+                                            ep['port'],
+                                            ep['auth']['plugin'],
+                                            { 'conf' : conf })
+                    except:
+                        print(f'Error initializing transport to ldmsd {dmn_name} at endpoint {ep_name} '\
```

Comment:
I wish that you print the Exception sting as well. I tested this with `ldms-test/maestro_cfg_test` and it failed. I had to use `pdb` to get to why it failed:
```python
Traceback (most recent call last):
  File "/opt/ovis/sbin/maestro", line 853, in <module>
    maestro = MaestroMonitor(etcd_spec, args)
  File "/opt/ovis/sbin/maestro", line 74, in __init__
    self.comms = self.build_comms()
  File "/opt/ovis/sbin/maestro", line 139, in build_comms
    ep['auth']['plugin'],
KeyError: 'plugin'
```

It would be nice to at the lease have "KeyError: 'plugin'" as a part of the error message in this case. I think catching the Exception as `e` and add "{e}" into the message should help.

Reply:
Good idea, I'll add the exception string to the output